src/compass/utils/age.py

```diff
@@ -323,7 +323,7 @@ def get_cslc(cslc_file, mission_id='S1', pol='VV'):
     '''
 
     if mission_id == 'S1':
-        cslc_path = f'science/SENTINEL1/CSLC/grids/{pol}'
+        cslc_path = f'CSLC/data/{pol}'
```

Comment:
```suggestion
        cslc_path = f'{DATA_PATH}/{pol}'
```
Add `DATA_PATH` to imports up top to reuse common path in `h5_helpers.py`.

Reply:
Where should I add `DATA_PATH` to make it an import?

Reply:
Below `from shapely import geometry` add:

```python

from compass.utils.h5_helpers import DATA_PATH
```